Library/Formula/knot.rb

```diff
@@ -39,6 +40,7 @@ def install
     inreplace "samples/Makefile", "install-data-local:", "disable-install-data-local:"
 
     system "make"
+    system "make", "check"
```

Comment:
Ok, seems fine, can you add comments elaborating on this? I meant let's add an option for `make check`.